tasks/configure-preprocessing-worker.yml

```diff
@@ -9,6 +9,17 @@
     mode: 0755
     state: "directory"
 
+- name: Copy the allowed file formats CSV file
+  template:
+    src: "{{ item }}"
+    dest: /etc/allowed_file_formats.csv
+    owner: root
+    group: root
+    mode: 0644
+  with_first_found:
+    - templates/{{ inventory_hostname }}/allowed_file_formats.csv.j2
```

Comment:
Does this allow to have a different CSV file in the playbooks and configure it from there?

Reply:
Yes, if a custom config for the hosts exists it will be used, else we use the default